packages/kit/test/apps/basics/test/test.js

```diff
@@ -1638,7 +1638,10 @@ test.describe('Load', () => {
 		}
 	});
 
-	test('using window.fetch causes a warning', async ({ page, javaScriptEnabled }) => {
+	test.only('using window.fetch causes a warning', async ({ page, javaScriptEnabled }) => {
```

Comment:
The `.only` causes CI to fail

```suggestion
	test('using window.fetch causes a warning', async ({ page, javaScriptEnabled }) => {
```